TTS/tts/datasets/dataset.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         outputs_per_step: int = 1,
         compute_linear_spec: bool = False,
         ap: AudioProcessor = None,
+        audio_config = None, # pylint: disable=unused-argument
```

Comment:
why do you need the `audio_config`? It is not being used here I guess

Reply:
I'm working on removing the args and using audio processor, there were some bugs making preprocessing and training slow but I've fixed them so now I just need to replace all audio config with audio processor

Reply:
fixed